src/core/graph.js

```diff
@@ -524,6 +684,14 @@ export class Graph {
     this._maybeCheckInvariants();
   }
 
+  /**
+   * Checks whether this Graph is equal to another Graph.
+   *
+   * Two Graphs are considered equal if they have identical sets of nodes
+   * and edges.
+   *
+   * Runs in O(n + e) where n is the number of nodes and e is the number of edges.
```

Comment:
```
   * Checks whether this Graph is equal to another Graph.
   *
   * Two Graphs are considered equal if they have identical sets of nodes
   * and edges.
```

Good. Could choose to explicitly call out that insertion order and
duplication are irrelevant.

```
   * Runs in O(n + e) where n is the number of nodes and e is the number of edges.
```

Specify “Runs in time `O(n + e)`” (or “worst-case time” if you prefer).
Add comma before “where”.